homeassistant/components/statistics/sensor.py

```diff
@@ -510,16 +511,16 @@ def _update_value(self):
 
         if self._state_characteristic not in STATS_NOT_A_NUMBER:
             with contextlib.suppress(TypeError):
-                value = round(value, self._precision)
+                value = round(cast(float, value), self._precision)
                 if self._precision == 0:
                     value = int(value)
         self._value = value
 
     # Statistics for numeric sensor
 
-    def _stat_average_linear(self):
+    def _stat_average_linear(self) -> StateType:
         if len(self.states) >= 2:
-            area = 0
+            area: float = 0
```

Comment:
Or do: `area = 0.0`

Reply:
I prefer over-typing to ensure specificity... Do you have a preference?

Reply:
No, it's fine as is